examples/skr/aks/unwrap.sh

```diff
@@ -20,7 +20,7 @@ if [ -z "${KEY_PROVIDER_PORT}" ]; then
   KEY_PROVIDER_PORT=50000
 fi
 
-AAA=`printf aasp | base64 -w0`
```

Comment:
Would doing this break things if they uploaded a key with the old version of the file then updated the container and tried to release the same key?

Reply:
So it wouldn't break key release since the Attestation Agent value doesn't have to do with the upload/claims. Also, the use of AASP is AKS SKR specific and as far as I know, not many customers are onboarded to it yet. So I figured we'd make this change now to avoid breaking more later and avoid the question of "what is AASP."